docs/getting-started/01-namespace.yaml

```diff
@@ -1,4 +0,0 @@
-apiVersion: v1
-kind: Namespace
```

Comment:
Deleted this -- the docs explicitly mention creating the namespace. The formerly commented out test was using this. I replaced it with a call to create the namespace in the test script.